### ISSUE TITLE
ValueError: not enough values to unpack (expected 2, got 0)

### ISSUE THREAD

**Issue by Zeyi-Lin** [Dai-Wenxun/MotionLCM#2]
Hey, when I reached the 🎬 Demo step, running the code reported the following error:

```
(motionlcm) linzeyi@trainer1:~/MotionLCM$ python demo.py --cfg configs/motionlcm_t2m.yaml --example assets/example.txt
05/01/2024 16:20:15 - INFO - __main__ - Loading checkpoints from experiments_t2m/motionlcm_humanml/motionlcm_humanml.ckpt
05/01/2024 16:20:15 - INFO - __main__ - Is LCM: True
05/01/2024 16:20:15 - INFO - __main__ - Is Controlnet: False
Traceback (most recent call last):
  File "/home/linzeyi/MotionLCM/demo.py", line 154, in <module>
    main()
  File "/home/linzeyi/MotionLCM/demo.py", line 74, in main
    datasets = get_datasets(cfg, phase="test")[0]
  File "/home/linzeyi/MotionLCM/mld/data/get_data.py", line 66, in get_datasets
    dataset = dataset_module_map[dataset_name.lower()](
  File "/home/linzeyi/MotionLCM/mld/data/HumanML3D.py", line 38, in __init__
    self._sample_set = self.get_sample_set(overrides=sample_overrides)
  File "/home/linzeyi/MotionLCM/mld/data/base.py", line 29, in get_sample_set
    return self.Dataset(split_file=split_file, **sample_params)
  File "/home/linzeyi/MotionLCM/mld/data/humanml/dataset.py", line 128, in __init__
    name_list, length_list = zip(
ValueError: not enough values to unpack (expected 2, got 0)
```

My Environment:
- OS: Linux-5.15.0-1036-oracle-x86_64-with-glibc2.31

I use the preparation process provided in the README and confirm that there are no errors.

**Comment by Dai-Wenxun** [comment]
Fixed in  449fa74cb491a7d08101f0ca16a85c8812e60052, thanks for reporting this bug.
![image](https://github.com/Dai-Wenxun/MotionLCM/assets/58767402/a5a01d0d-0939-4bcc-bdda-0e5eb7d55f1d)

**Comment by Zeyi-Lin** [comment]
> Fixed in [449fa74](https://github.com/Dai-Wenxun/MotionLCM/commit/449fa74cb491a7d08101f0ca16a85c8812e60052), thanks for reporting this bug. ![image](https://private-user-images.githubusercontent.com/58767402/327088299-a5a01d0d-0939-4bcc-bdda-0e5eb7d55f1d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ1NTYzNTksIm5iZiI6MTcxNDU1NjA1OSwicGF0aCI6Ii81ODc2NzQwMi8zMjcwODgyOTktYTVhMDFkMGQtMDkzOS00YmNjLWJkZGEtMGU1ZWI3ZDU1ZjFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAxVDA5MzQxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4NGJkMTA5YzRiZDhmZTVkZjlmMWM2ZjJmODMwMzY0OTQxMjIzZDBjOGEyMThiYmE1YjBmODM3MmNiY2ZjMDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ok65I8wMvJ-vbmE1x9zgJzKwymuBY3oh_YnOX3i7jjI)

Thanks —— The reason for the error is that I directly copied the `HumanML3D` folder from [HumanML3D](https://github.com/EricGuo5513/HumanML3D) to the `datasets` directory, but in fact, the processing workflow needs to be executed.

🎉The code in the red box solved my problem.
